### PR TITLE
PivotGrid.ExcelJS export - support the predefined date formats

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by novsstation — DISMISSED** [review]

**Review by jsdmitry — DISMISSED** [review]

File: js/exporter/exceljs/export_pivot_grid.js [hunk 1]
```diff
@@ -153,6 +157,10 @@ function _exportRow(rowIndex, cellCount, row, startColumnIndex, dataProvider, cu
     }
 }
 
+function _convertDateForExcelJS(date) {
+    return new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate(), date.getHours(), date.getMinutes(), date.getSeconds(), date.getMilliseconds()));
```
Comment (IgnatovDan) [inline js/exporter/exceljs/export_pivot_grid.js:161]: https://github.com/DevExpress/DevExtreme/blob/20_2/js/exporter/exceljs/export_data_grid.js#L178 - the same function
'utils'?

File: testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js [hunk 2]
```diff
@@ -1616,6 +1616,92 @@ QUnit.module('Scenarios', moduleConfig, () => {
...
+            { value: dateValue.getTime(), expectedPivotCellValue: dateValue, expectedExcelCellValue: new Date(Date.UTC(2019, 9, 9, 9, 9, 9, 9)) }
+        ].forEach((date) => {
+            QUnit.test(`Export [string x string x number] with date format '${format.format}', cell.value: ${JSON.stringify(date.value)}`, function(assert) {
+                const done = assert.async();
+
+                const ds = {
+                    fields: [
+                        { area: 'row', dataField: 'row1', dataType: 'string' },
+                        { area: 'column', dataField: 'col1', dataType: 'string' },
+                        { area: 'data', summaryType: 'max', dataType: 'date', dataField: 'date', format: format.format }
```
Comment (IgnatovDan) [inline testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js]: What about 'date' type in column/row areas?

                        { area: 'row', dataField: 'row1', dataType: 'date' },
                        { area: 'column', dataField: 'col1', dataType: 'date' },

Can it be shown as a 'simple date' without Y/Q/M ?

File: js/exporter/exceljs/export_pivot_grid.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply js/exporter/exceljs/export_pivot_grid.js:161]: We have a separate task for allocation of general parts for DataGrid & PivotGrid export. We are going to do it later.

File: testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js (hunk 2 above)
Comment (EugeniyKiyashko) [reply testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js]: I checked it. 'Row' and 'Column' areas support the predefined custom format too. There are additional tests scenarios into the merge. Please, look at again

**Review by novsstation — APPROVED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**